t/02-bowels/80-repo-lock.t

```diff
@@ -113,4 +114,29 @@ my $t = Pinto::Tester->new;
 }
 
 #------------------------------------------------------------------------------
+subtest 'Test stale lock file' => sub {
+
+    # create dummy lock file not connected to us
+    my $lockfile = $t->root->file('.lock');
+    my $fh = IO::File->new( $lockfile, 'w' );
```

Comment:
no specific reason for IO::File, my first encounter with Path::Class::File and didn't dig deep enough, pushed the correction